src/analysis/normed_space/star/basic.lean

```diff
@@ -105,23 +105,17 @@ instance ring_hom_isometric.star_ring_end [normed_comm_ring E] [star_ring E]
 namespace cstar_ring
 variables [normed_ring E] [star_ring E] [cstar_ring E]
 
+lemma norm_star_le {x : E} (htriv : x ≠ 0) : ∥x⋆∥ ≤ ∥x∥ :=
+le_of_mul_le_mul_right (by simp [← norm_star_mul_self, norm_mul_le])
+  (norm_pos_iff.mpr ((add_equiv.map_ne_zero_iff star_add_equiv).mpr htriv))
```

Comment:
I think this probably just belongs as a `have` inside `to_normed_star_monoid`; `htriv` is a useless assumption.

Reply:
Also, isn't `add_equiv.map_ne_zero_iff star_add_equiv` precisely `star_ne_zero` from above?

Reply:
You are right about both.  I removed the assumption `htriv` and the lemma is a have in the proof of the instance `to_normed_star_monoid`.